utils/src/logger.ts

```diff
@@ -0,0 +1,28 @@
+import { styleText } from "node:util";
+
+export function info(codemodName: string, ...args: unknown[]) {
+  console.info(styleText(["bold", 'cyan'], `[${codemodName}]`), ...args);
+}
+
+export function warn(codemodName: string, ...args: unknown[]) {
+	  console.warn(styleText(["bold", 'yellow'], `[${codemodName}]`), ...args);
+}
+
+export function error(codemodName: string, ...args: unknown[]) {
+  console.error(styleText(["bold", 'red'], `[${codemodName}]`), ...args);
+}
+
+export function debug(codemodName: string, ...args: unknown[]) {
+  if (process.env.DEBUG) {
+	console.debug(styleText(["bold", 'magenta'], `[${codemodName}]`), ...args);
+  }
+}
+
+const logger = {
+	info,
+	warn,
+	error,
+	debug,
+};
+
+export default logger;
```

Comment:
We can improve this further via a createLogFunction(colors, level, name! helper, and add support for the LOGLEVEL environment variable.

Reply:
I wonder if we really need something so complicated. With this simple little code here, we cover 99% of our use cases. Maybe we can improve it when we need it for future codemodes.

Reply:
I don't think it's that complicated, and it's DRY (Don't repeat yourself)

Reply:
👀